package.json

```diff
@@ -12,10 +12,15 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@emurgo/cardano-serialization-lib-nodejs": "^9.1.2",
     "@cardano-ogmios/client": "^4.2.1",
     "big-integer": "^1.6.51",
     "bufferutil": "^4.0.5",
     "uniqid": "^5.4.0",
     "ws": "^8.4.0"
+  },
```

Comment:
Could you remove these please? Perhaps this is from an outdated version of @adamczykm branch. These should all be provided by Nix